examples/mplot3d/rotate_axes3d_demo.py

```diff
@@ -1,13 +1,21 @@
+'''
+A very simple animation of a rotating 3D plot.
+
+See wire3d_animation_demo for another simple example of animating a 3D plot.
+'''
+
 from mpl_toolkits.mplot3d import axes3d
 import matplotlib.pyplot as plt
-import numpy as np
-
 
 fig = plt.figure()
 ax = fig.add_subplot(111, projection='3d')
+
+# load some test data for demonstration and plot a wireframe
 X, Y, Z = axes3d.get_test_data(0.1)
 ax.plot_wireframe(X, Y, Z, rstride=5, cstride=5)
 
+# rotate the axes and update
 for angle in range(0, 360):
     ax.view_init(30, angle)
     plt.draw()
+    plt.pause(.001)
```

Comment:
On my computer, this line was needed to make the example viewable.  I got the idea for it when I did the wire3d_animation_demo example, but I'm not sure this is the correct practice since in both examples this line seems to trigger a deprecation warning: 
```
/Users/trish/anaconda/lib/python2.7/site-packages/matplotlib-2.0.0b1+1720.ga8ad349.dirty-py2.7-macosx-10.5-x86_64.egg/matplotlib/backend_bases.py:2444: MatplotlibDeprecationWarning: Using default event loop until function specific to this GUI is implemented
  warnings.warn(str, mplDeprecation)
```
Let me know if there's something else I should do with this instead.